cache/edge_lambdas/toggler.js

```diff
@@ -4,20 +4,20 @@
 // {
 //   id: 'outro',
 //   title: 'Outro',
-//   range: [0, 50], // 50% chance, this is [inclusive, exclusive)
-//   range: [50, 100], // 50% chance, this is [inclusive, exclusive)
-//   shouldRun: (request, range) => {
+//   range: [0, 50], // Run this test on 50% of users
+//   range: [50, 100], // Run this test on 50% of users - not overlapping with ☝️
+//   when: (request, range) => {
 //     return request.uri.match(/^\/articles\/*/);
 //   }
 // }
 
 // This is mutable for testing
 let tests = [
   {
-    id: 'searchUsingScoringTiers',
+    id: 'testg',
```

Comment:
?

Reply:
Nice catch - thanks, we were using this for an illustration of how it would work.